CONTRIBUTING.txt

```diff
@@ -379,6 +379,27 @@ To rebuild a full clean documentation, run:
     make clean
     make html
 
+Gallery
+~~~~~~~
+
+If you are contributing an example to the
+`gallery <https://scikit-image.org/docs/dev/auto_examples/>`_ (or editing an
+existing one), write in
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_.
+Indeed, the gallery is a
+`Sphinx-Gallery <https://sphinx-gallery.github.io/stable/index.html>`_ one.
+
+Refer to the above to build (all) the docs, and check how your edits render at
```

Comment:
Maybe mention the browser here, otherwise newcomers might not know how to check.

Reply:
Good point, realized that when replying: https://github.com/scikit-image/scikit-image/pull/4851#issuecomment-663026833
Done bdfbbd245.